canonical-web-and-design/maas-ui
# Add Google Analytics script

Done:
- Conditionally include the Google Analytics code when enabled. Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/347.

QA:
- Run the app with the `NODE_ENV` set to something other than 'development' (or change the two places that check for debug mode).
- In the general settings make sure google analytics is enabled.
- Inspect the page and make sure there is a script tag immediately above the `p-navigation` header.
- Turn off the google analytics option in the settings and the script tag is gone.
- Do this check for both the React and Angular clients.
